syntaxes/src/build.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import * as fs from 'fs';
+
+import {GrammarDefinition, GrammarDefinitionValue, JsonObject, JsonObjectValue} from './types';
+
+import {template} from './template';
+
+const processValue = (value: GrammarDefinitionValue): JsonObjectValue => {
+  if (typeof value === 'string') {
+    return value;
+  } else if (value instanceof RegExp) {
+    return value.toString().replace(/^\/|\/$/g, '');
```

Comment:
why is this done?

Reply:
- `RegExp.prototype.toString` will do the [escaping](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/toString#Empty_regular_expressions_and_escaping) we need in the output file, while feeding the regex literals directly to `JSON.stringify` will not
- `RegExp.prototype.toString` retains the demarcating `/` characters, so they need to be trimmed from the final output

Reply:
Sorry, I meant to ask why the replacement?

Reply:
Added an inline comment to clarify this in the code